deis-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -17,4 +17,5 @@ data: {{ if eq .storage "gcs" }}
   accesskey: {{ b64enc .s3.accesskey }}
   secretkey: {{ b64enc .s3.secretkey }}
   region: {{ b64enc .s3.region }}
-  bucket: {{ b64enc .s3.bucket }}{{ end }}
+  bucket: {{ b64enc .s3.bucket }}
+  database-bucket: {{ b64enc .s3.database-bucket }}{{ end }}
```

Comment:
Thought about it..but @mboersma has said that there may be few users who would want them in separate buckets